corehq/apps/enterprise/enterprise.py

```diff
@@ -242,10 +242,12 @@ def rows_for_domain(self, domain_obj):
         apps = {a.id: a.name for a in apps}
         rows = []
         for hit in self.hits(domain_obj.name):
+            if hit['form'].get('#type') == 'system':
```

Comment:
```suggestion
            if "#type" in hit['form'].keys() and hit['form'].get('#type') == 'system':
```
Would something like this make sense? Or if type isn't on the form would we want it to fail hard?

Reply:
`.get()` will return `None` if it's not there.

Reply:
Ah of course